src/raster/r.maxent.train/r.maxent.train.html

```diff
@@ -0,0 +1,91 @@
+another of background points locations. Both need to include columns 
+with the X, Y and sample values of the environmental variables that you 
+want to use as predictor variables. You can use the <a 
+href="r.out.maxent_swd.html">r.out.maxent_swd</a> or <a 
+href="v.maxent.swd.html">v.maxent.swd</a> addons to create these files. 
+For more details about the structure of these files, see the <a 
+href="https://biodiversityinformatics.amnh.org/open_source/maxent/">Maxent 
+website</a>.
+
+<p>
+The only other requirement is to provide an output folder. With 
+these inputs, a Maxent model will be created. If you also provide a 
+folder with environmental raster layers with names corresponding to the 
+names of the environmental variables in the SWD files, the module will 
+create a prediction (suitability distribution) raster layer as well. 
+
+<p>
+Note that the Maxent software generates ascii files without projection 
+information. That means you need to make sure yourself that the 
+environmental layers you provide are in the same reference coordinate 
+system as your current mapset. 
+
+<p>
+The addon provides access to nearly all parameters available in the 
+Maxent software. On the above-mentioned website, you can find a 
+tutorial that explains most of these options. For the other options, 
+see the Maxent help file.
+
+
+<h2>NOTES</h2>
+
+<p>
+The addon requires the Maxent software. You can download the software 
+from the <a 
+href="http://biodiversityinformatics.amnh.org/open_source/maxent">Maxent 
+website</a>. The software includes a <em>Maxent.jar</em> file. You can 
+simply point to that file (<em>maxent</em> options) when using the 
+addon. If you plan to use the addon more often, the easier solution 
+will be to copy the Maxent.jar file to the addon/script directory. On 
+GNU/Linux, the default directory is $HOME/.grass8/addons. On MS-Windows 
+it is %APPDATA%\Roaming\GRASS8\addons. The name of the directory is 
+stored in the GRASS_ADDON_BASE environmental variable. Alternatively, 
+open GRASS GIS AND type `echo $GRASS_ADDON_BASE` on the command line. 
+This will give you the location of the default addon directory on your 
+Computer.
+
+A new addon is being developed that 
+will allow you to use the model to performs predictions based on 
+different sets of prediction layers. For now, check the Maxent tutorial 
+for instructions on how you can use Maxent itself to perform predictions 
+based on a set of prediction layers. Note that the <em>v.maxent.swd</em> 
+addons provides an easy way to export the prediction layers in the 
+right format.
+
+<h2>REFERENCES</h2>
+
+<ul>
+<li>Steven J. Phillips, Miroslav Dud&iacute;k, Robert E. Schapire. 2020: Maxent software
+for modeling species niches and distributions (Version 3.4.1).
+Available from url: <a href="http://biodiversityinformatics.amnh.org/open_source/maxent">
+http://biodiversityinformatics.amnh.org/open_source/maxent</a>
```

Comment:
```suggestion
Available from url: <a href="https://biodiversityinformatics.amnh.org/open_source/maxent">
https://biodiversityinformatics.amnh.org/open_source/maxent</a>
```
I checked that the https link is valid